src/rulesetDefinitionChecker.ts

```diff
@@ -97,10 +97,6 @@ export class RulesetDefinitionChecker {
     };
 
     private typeLinks: {[key: string]: string[]} = {
-        'armors.corpseBattle': ['items'],
-        'alienDeployments.data[].itemSets[]': ['items'], // could check if it's a weapon?
```

Comment:
@Finnik723 do I add this one back in? Or was it incorrect?

Reply:
It can be any item, im using that to give extra mission loot